packages/web-client/app/resources/safes.ts

```diff
@@ -195,8 +196,27 @@ export class Safes extends Resource<Args> {
           individualUpdate.blockNumber > this.graphData.blockNumber
             ? individualUpdate.safe
             : graphDataSafe;
+      } else if (graphDataSafe) {
+        selectedSafe = graphDataSafe;
+      } else if (individualUpdate) {
+        selectedSafe = individualUpdate.safe;
       } else {
-        selectedSafe = graphDataSafe ?? individualUpdate.safe;
+        let e = new Error(
+          `Safe with address "${address}" was not found in graph data or individual update data`
+        );
+
+        Sentry.withScope((scope) => {
+          // we want to know if there are actually any addresses available
+          scope.setExtras({
```

Comment:
This is deprecated but still works, and allows a bigger payload (16kb) than the alternative (8kb)